src/plugins/stdioproc/README.md

```diff
@@ -0,0 +1,201 @@
+- infos = Information about the stdioproc plugin is in keys below
+- infos/author = Klemens Böswirth <k.boeswirth+git@gmail.com>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides =
+- infos/recommends =
+- infos/placements = postgetstorage presetstorage
+- infos/status = maintained unittest shelltest experimental
+- infos/metadata =
+- infos/description = one-line description of stdioproc
+
+## Introduction
+
+This plugin spawns a new process with a user-defined executable and delegates all operations to the new process.
+
+## Usage
+
+Set the config key `app` and the arrays `args/#`, `env/#` to the path of an executable, the arguments that shall be passed and the environment variables to be set.
+
+```
+kdb mount test.dump /tests/stdioproc stdioproc 'app=/usr/bin/pluginproc' 'args=#1' 'args/#0=--load-plugin' 'args/#1=myplugin'
```

Comment:
Do we provide this pluginproc as example?

Reply:
No, this is a purely hypothetical example. A real world example (Java) can be found below. I didn't want to use the `testapp.sh` as an example, since it doesn't really do what the child process is "supposed to". It only does things that make it convenient for testing.